xml/security_ldap.xml

```diff
@@ -861,6 +788,99 @@ systemctl start sssd</screen>
  </sect2>
 </sect1>
 
+<!-- default installation uses self-signed cert -->
+ <sect1 xml:id="sec-security-ldap-server-ca">
+ <title>Using CA Certificates for TSL</title>
+ <para>
+  You can manage the CA certificates for &ds389; with the following command
+  line tools: <command>certutil</command>, <command>openssl</command>, and
+  <command>pk12util</command>.
+ </para>
+ <para>
+  For testing purposes, you can create a self-signed certificate with
+  <command>dscreate</command>. Find the certificate at
+  <filename>/etc/dirsrv/slapd-localhost/ca.crt</filename>. For remote administration,
+  copy the certificate to a readable location. For production environments,
+  contact a CA authority of your organization's choice and request a server
+  certificate, a client certificate, and a root certificate.
+ </para>
+ <para>
+ Make sure to meet the following requirements before executing the procedure below:
+ </para>
+ <itemizedlist>
+  <listitem>
+   <para>
+    You have a server certificate and a private key to use for the TSL connection.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    You have set up an NSS (Network Security Services) database (for example,
+    with the <command>certutil</command> command).
+   </para>
+  </listitem>
+ </itemizedlist>
+
+ <procedure>
+  <para>
+  Before you can import an existing private key and certificate into the NSS
+  (Network Security Services) database, you need to create a bundle of the
+  private key and the server certificate. This results in a <filename>*.p12</filename>
+  file.
+  </para>
+  <important>
+  <title><filename>*.p12</filename> File and Friendly Name</title>
```

Comment:
Sorry to be that guy bbbbuuuututtttttt you could rewrite this in terms of dsctl instance tls commands rather than the direct openssl/p12/certutil commands.

Reply:
Also, if we do want the openssl/p12 commands, this is an older guide I wrote, but probably the most "up to date" tls docs https://www.port389.org/docs/389ds/howto/howto-ssl.html

Reply:
Yes willdolater :)